developer_setup/seeding_test_inventory.md

```diff
@@ -0,0 +1,69 @@
+## Seeding Test Inventory
```

Comment:
Probably a better name for this

Reply:
Using test recordings to populate a development database